lib/runner.js

```diff
@@ -57,6 +57,10 @@ optionChain(chainableMethods, function (opts, title, fn) {
 		title = null;
 	}
 
+	if (opts.todo && typeof fn === 'function') {
+		throw new TypeError('`todo` tests are not allowed to have a function. Use `test.skip()` for tests with a function.');
+	}
+
```

Comment:
That's what I was doing before but it wasn't working because of the `fn = noop` line (which seems to be used to "override" any given function), and if I removed that other stuff broke.